### ISSUE TITLE
Conan command parameter expansion from file content

### ISSUE THREAD

**Issue by lasote** [conan-io/conan#5743]
Allow an special pattern like `f(path/to/file)` (format to be defined) to be used in the "conan upload" and others probably. Conan will automatically read the contents of the file an will apply it as an argument.

Make sense together with: https://github.com/conan-io/conan/issues/5744

**Comment by lasote** [comment]
The `(` are also problematic. I suggest: `@path@` (we cannot do just starting with @ because it could collide with what we treat as a reference)

**Comment by lasote** [comment]
Waiting for #5744

**Comment by lasote** [comment]
Looks like this could go in the direction of accepting lock files, maybe also to allow uploading several references. Assigning to the next release because it is not fully clear what do we want to do.

**Comment by Sebiee** [comment]
> maybe also to allow uploading several references

I'm very interested in such a feature.

My use-case is that I would like to upload only the packages that were built during the install phase (used `-b missing`). I managed to obtain a list of those packages using `conan info -b missing` but it seems there's no way to do `conan upload 'boost/*,zlib/*,libressl/*` for example.

I could of course do a separate `conan upload` instruction for each reference, but this would make the `--parallel` parameter of no use to me. I would like to take advantage of that feature for faster uploading too.

Until now, I used to always do `conan upload '*'` after `conan install --build missing` but I really want to get rid of this since the upload command would take a lot of time (because it is compressing large deps like boost) even if no upload would need to be made.
